presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -146,6 +146,15 @@ class TaskManager {
   // in exec/Task.h).
   std::array<size_t, 5> getTaskNumbers(size_t& numTasks) const;
 
+  /// Populate the blocked tasks (failing to take lock on the mutex), and long
+  /// running operator calls across all drivers in all tasks.  Returns false if
+  /// a lock on the taskMap cannot be taken, otherwise returns true.
+  bool getLongRunningOpCalls(
+      std::chrono::nanoseconds lockTimeout,
+      size_t thresholdDurationMs,
```

Comment:
We currently use the same timeout for both of them.
Does it make sense to only hav one function argument for it?